includes/class-convertkit-mm-actions.php

```diff
@@ -78,17 +78,18 @@ public function add_member( $member_data ) {
 		}
 
 		// Fetch data from member array.
-		$user_email = $member_data['email'];
-		$first_name = rawurlencode( $member_data['first_name'] );
```

Comment:
We don't need to `rawurlencode` names, as the API class will handle encoding spaces.